projects/templates/projects/project_list.html

```diff
@@ -4,11 +4,12 @@
 {% endblock %}
 
 {% block content %}
-{% for project in object_list %}
     <div class="content-section-a">
 
         <div class="container">
             <div class="row">
+                <a href="upload/">Upload a project</a>
```

Comment:
Why "upload a project" it is "create project". You can upload a file not a project.